libzecale/zecale_constants.hpp

```diff
@@ -2,16 +2,21 @@
 //
 // SPDX-License-Identifier: LGPL-3.0+
 
-#ifndef __ZECALE_ZECALE_CONSTANTS__
-#define __ZECALE_ZECALE_CONSTANTS__
+#ifndef __ZECALE_ZECALE_CONSTANTS_HPP__
```

Comment:
libzecale should probably be completely agnostic to these constants, so maybe these constants would be better in the aggregator_server directory?

Reply:
I followed a similar approach to what we did in Zeth: https://github.com/clearmatics/zeth/blob/develop/libzeth/zeth_constants.hpp here.
I am happy to define these directly in the aggregator_server, but I would be as consistent as possible w.r.t. these things across our projects, so that would need to be reflected in the Zeth-side too.

Reply:
TBH - I would almost be tempted to move these to the CMakeLists file (along with the curve selection) to define the protocol params in the build config...

Reply:
Yes, makes sense to move those 3 constants in zeth as well.

No problem with the batch size etc being defined by CMake if thats easy enough.  It would be ideal if they were out of scope during the build of libzecale (similaly for libzeth), i.e. it could be good to put them somewhere other than build/zecale.h.  But that can be fixed up later if it's awkward.

Reply:
As noted offline by @AntoineRondelet, `libzecale::ZECALE_NUM_INPUTS_PER_NESTED_PROOF` is not necessarily a simple independent constant. I'll create an issue to resolve this in the future.

Reply:
> I'll create an issue to resolve this in the future.
Thanks @dtebbs
As per our chat, I dropped the commit that used the `libzecale::` constants in the aggregator_server to avoid "non-trivial" changes before v0.4. This reverts to the use of the `static const`s in the aggregator_server.cpp. I also removed the `libzecale/zecale_constants.hpp` file which was unused as of now. Let's see if we need to re-introduce it later on.

Reply:
Great.  For now, I've created #79 to resolve this later.